cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/utils/CobiGenUtils.java

```diff
@@ -60,6 +60,22 @@ public static CobiGen initializeCobiGen(Path templatesProject) {
     }
   }
 
+  /**
+   * Registers CobiGen plug-ins and instantiates CobiGen, even if old templates found
```

Comment:
Better: Registers CobiGen plug-ins and instantiates CobiGen, even if monolithic templates were found